chart/epinio-installer/templates/NOTES.txt

```diff
@@ -6,8 +6,9 @@ You can get the IP address of your Ingress controller by passing this command:
 Make sure your domain points to the IP address of your Ingress controller (1.2.3.4).
 {{ end }}
 To interact with your Epinio installation, you have to download the latest epinio binary https://github.com/epinio/epinio/releases.
```

Comment:
`you have to` sound more`it is mandatory`, this maybe better to use replace the last one with something like `you could`?